components/contribution-flow/CollectiveTitleContainer.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import Container from '../Container';
+import Link from '../Link';
+
+const CollectiveTitleLink = styled(Link)`
+  color: ${props => props.linkColor};
+  cursor: pointer;
+`;
+
+const CollectiveTitleContainer = ({ collective, useLink, children, linkColor }) => {
+  if (useLink) {
+    return (
+      <CollectiveTitleLink href={`/${collective.slug}`} linkColor={linkColor}>
```

Comment:
:ice_cream: Simpler alternative

```suggestion
      <StyledLink as={Link} href={`/${collective.slug}`} color={linkColor}>
```

Reply:
Done. Thanks for pointing that out. 😄